### PR TITLE
Added Kerr-Newman limits

### LINKED ISSUE #109: Coding the right event horizon for Kerr & Kerr-Newman space-time
🐞 **Problem**
As of now, in calculation of time-like trajectory in Kerr metric stops and raises a warning on reaching schwarzschild radius. But event horizon in kerr spacetime is smaller than schwarzschild radius. But the exact values need to be researched.

🎯 **Goal**
  - Research about the problem and specify the correct radius after which computation should stop in `metric/kerr.py`.

- [x] Kerr

- [ ] Kerr-Newman

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #109`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by obi-wan-shinobi**
Added calculations for event horizon and ergosphere of a Kerr-Newman black hole.

Fixes #109 

@ritzvik 
@shreyasbapat 

Sorry about a new PR. I guess I'm still not that used to git :0

### CODE REVIEW

File: src/einsteinpy/utils/kerrnewman_utils.py [hunk 1]
```diff
@@ -561,3 +563,95 @@ def kerrnewman_time_velocity(pos_vec, vel_vec, mass, a, Q):
...
+        Mass of massive body
+    a : float
+        Black hole spin factor
+    Q : float
+        Charge on the black hole
+    theta : float
+        Angle from z-axis in Boyer-Lindquist coordinates in radians. Mandatory for coord=='Spherical'. Defaults to pi/2.
+    coord : str
+        Output coordinate system. 'BL' for Boyer-Lindquist & 'Spherical' for spherical. Defaults to 'BL'.
+
```
Comment (ritzvik) [inline src/einsteinpy/utils/kerrnewman_utils.py]: Kindly add the docstrings for `c`,`G` and `Cc`. Please do the same for the other function and in `kerr_utils.py`

**Comment by ritzvik** [comment]
Tests would most likely be ignored for this, I'll decide upon it

File: src/einsteinpy/utils/kerrnewman_utils.py (hunk 1 above)
Comment (obi-wan-shinobi) [reply src/einsteinpy/utils/kerrnewman_utils.py]: Okay.

**Comment by obi-wan-shinobi** [comment]
Could you also review on the code coverage problem? I can't seem to find what's wrong... sorry :(

**Comment by ritzvik** [comment]
> Could you also review the code coverage problem? I can't seem to find what's wrong... sorry :(

I’ll get back to it.

File: src/einsteinpy/tests/test_metric/test_kerrnewman.py [hunk 2]
```diff
@@ -162,3 +167,25 @@ def test_calculate_trajectory_iterator_RuntimeWarning():
...
+
+
+def test_event_horizon_for_nonrotating_uncharged_case():
+    M = 5e27
+    a = 0.0
+    Q = 0.0
+    _scr = schwarzschild_radius_dimensionless(M)
+    a1 = kerrnewman_utils.event_horizon(M, a, Q, np.pi / 4, "Spherical")
+    a2 = kerr_utils.event_horizon(M, a, np.pi / 4)
+    assert_allclose(a1, a2, rtol=0.0, atol=1e-4)
```
Comment (ritzvik) [inline src/einsteinpy/tests/test_metric/test_kerrnewman.py]: The tests seems fine,
set `rtol=1e-4` and `atol=0` for every new test you added.

Comment (ritzvik) [reply src/einsteinpy/tests/test_metric/test_kerrnewman.py]: Also move the tests to `test_utils/test_kerrnewman_utils.py`

Comment (ritzvik) [reply src/einsteinpy/tests/test_metric/test_kerrnewman.py]: Last thing, take care of this `rtol`, `atol` business in the same tests of `test_utils/test_kerr_utils.py`

**Comment by shreyasbapat** [comment]
Pay attention to the commit author, if I am right, there's some discrepancy. Or maybe it's GitHub mobile app

File: src/einsteinpy/tests/test_metric/test_kerrnewman.py (hunk 2 above)
Comment (obi-wan-shinobi) [reply src/einsteinpy/tests/test_metric/test_kerrnewman.py]: okay, on it! :)

**Comment by obi-wan-shinobi** [comment]
> Pay attention to the commit author, if I am right, there's some discrepancy. Or maybe it's GitHub mobile app

I'm not sure I understand 😅

**Comment by ritzvik** [comment]
Everything looks good!
@obi-wan-shinobi Thanks !!

**Pull request merged**